### DISCUSSION TITLE
Experienced a disruption in camera connectivity and encountered a loss of a package.

### DISCUSSION THREAD

**Discussion by tomasenskede** [jomjol/AI-on-the-edge-device#2803] [Q&A]
Initially, I established a connection with the camera, but suddenly, the connection was lost. Upon pinging the ESP23-CAM, I observed a loss of TCP packages. Despite reflashing the ESP and recopying the latest files to the SD card, the issue persists. The WiFi signal strength is reported as fair (-76dBm). What could be causing this problem?
![IMG_5668](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/c7c121e4-fc8b-4c79-862d-360b3f036e5d)

**Comment by caco3** [comment]
Maybe there are some interferences, see https://github.com/jomjol/AI-on-the-edge-device/discussions/2551#discussioncomment-6607170.

Also people found that some wifi channels cause issues, so try channel 1, 6 or 11, see https://github.com/jomjol/AI-on-the-edge-device/discussions/2551#discussioncomment-6607170

**Reply by caco3** [reply]
see also https://github.com/jomjol/AI-on-the-edge-device/discussions/2406#discussioncomment-8348127
